openmdao/core/total_jac.py

```diff
@@ -1557,6 +1557,7 @@ def compute_totals_approx(self, initialize=False, progress_out_stream=None):
         model = self.model
         comm = model.comm
         return_format = self.return_format
+        debug_print = self.debug_print
```

Comment:
I notice that in compute_totals, setting debug_print also prints the amount of time taken to do the linear solves.  Would it be useful to print the time taken to linearize in compute_approx_totals?

Reply:
That could be useful. Let me try to add that in and fix the pep8 issues.